.cirrus.yml

```diff
@@ -196,6 +196,11 @@ task:
   name: 'no wallet, libbitcoinkernel'
   << : *GLOBAL_TASK_TEMPLATE
   persistent_worker:
+    isolation:
+      vetu:
```

Comment:
I wonder if this will make it impossible to run the CI locally without vetu?

Reply:
Not sure how others (sjors, inquistion) are running the CI

Reply:
(we use the raw scripts and a few env vars)